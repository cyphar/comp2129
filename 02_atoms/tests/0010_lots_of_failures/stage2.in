START 5 8 30
PLACE -1 0
PLACE -51 -2
PLACE 0 -1
PLACE 23 -5
PLACE 7 30
PLACE 100 -2
PLACE 7-2 27+4
PLACE 7+2 22
PLACE 7+2 22  a
PLACE 4 a 2 t
PLACE 4
PLACE
