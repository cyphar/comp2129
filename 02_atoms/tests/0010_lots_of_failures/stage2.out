Game Ready
Red's Turn

Invalid Coordinates

Invalid Coordinates

Invalid Coordinates

Invalid Coordinates

Invalid Coordinates

Invalid Coordinates

Invalid Coordinates

Invalid Coordinates

Too Many Arguments

Too Many Arguments

Missing Argument

Missing Argument

